src/ua/artcode/udiary/controller/MainControllerImpl.java

```diff
@@ -4,34 +4,51 @@
 import ua.artcode.udiary.dao.RecordDao;
 import ua.artcode.udiary.dao.UserDao;
 import ua.artcode.udiary.exception.AppException;
-import ua.artcode.udiary.model.Dairy;
-import ua.artcode.udiary.model.Record;
-import ua.artcode.udiary.model.User;
+import ua.artcode.udiary.model.*;
```

Comment:
I suggest avoiding an importing all from a package. It may confuse in some cases

Reply:
Fixed. Please check new version in the next push.